src/language-service/src/schemas/integrations/triggers.ts

```diff
@@ -191,13 +191,13 @@ interface NumericStateTrigger {
    * Fire this trigger if the numeric state of the monitored entity (or entities) is changing from above to below the given threshold.
    * https://www.home-assistant.io/docs/automation/trigger/#numeric-state-trigger
    */
-  below?: number;
+  below?: string | number;
```

Comment:
It is explicitly limited to `input_number` entities. The schema should reflect that. Right now, any string is accepted, which is not the case upstream.

Reply:
Done.

Reply:
Or maybe not.